tests/rebin_test.py

```diff
@@ -8,6 +8,7 @@
 # ----------------------------------------------
 #         Test Rebinning
 # ----------------------------------------------
+# TODO this only tests np.array inputs now but not xr.DataArray as inputs
```

Comment:
This will require `xarray` in `requirements-dev.txt`

Reply:
Removed it. Missed that one

Reply:
Can you remove the TODO comment as well?